### PR TITLE
Ansible namespace support flow

### PR DESCRIPTION
**Pull request by Hyperkid123**
Jira: https://issues.redhat.com/browse/RHCLOUD-32338

In addition to the required changes, I've also done some extra stuff. Let me know if you want to split it into a different PR
- added some notes when using the apple M chip, the usual project init did not work for me and some extra stuff had to be added
- also started using images for postgres from quay, because of docker hub rate limiting

### CODE REVIEW

**Comment by tahmidefaz** [comment]
@Hyperkid123 Looking good!
I'd recommend the following:
* Add a test for the flow. Some examples: [Here](https://github.com/RedHatInsights/astro-virtual-assistant/tree/main/tests/stories)
* Saw a number of steps missing from what we have on Miro. Any reason why?

**Comment by justinorringer** [comment]
>     * Saw a number of steps missing from what we have on Miro. Any reason why?
Looks like they canned the fun one (its half baked anyway). He chose the right one

File: data/ansible/nlu.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: "3.1"
+
+nlu:
+  - intent: intent_ansible_namespaces_access_issues
+    examples: |
+      - unable to access my ansible namespace
+      - unable to get to my ansible namespace
+      - i used to be able to access my ansible namespace but i can't anymore
+      - are you able to get me access to a ansible namespace
+      - it says i don't have access to my ansible namespace
```
Comment (justinorringer) [inline data/ansible/nlu.yml:10]:
```suggestion
      - it says i don't have access to my ansible namespace
      - ansible namespace access
      - ansible namespace issues
      - ansible namespace problems
      - ansible namespace doesn't work
      - how do I get access to an ansible namespace
      - fix ansible namespace 
```

Our bot is dumb, some more examples

File: data/ansible/domain.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - intent_ansible_namespaces_access_issues
+
+entities: []
+
+actions: []
+
+slots: {}
+
+responses:
+  utter_inform_reach_out_to_support:
```
Comment (justinorringer) [inline data/ansible/domain.yml:13]:
```suggestion
  utter_ansible_inform_reach_out_to_support:
```

Just want to scope these. I don't know we would manage to duplicate the name elsewhere, but you never know

File: data/ansible/rules.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+version: "3.1"
+
+rules:
+  - rule: Reach out to support
+    steps:
+      - intent: intent_ansible_namespaces_access_issues
+      - action: utter_inform_reach_out_to_support
```
Comment (justinorringer) [inline data/ansible/rules.yml:7]:
```suggestion
      - action: utter_ansible_inform_reach_out_to_support
```

**Review by justinorringer — APPROVED** [review]
LGTM! Thanks for wading through the mac nonsense

**Comment by Hyperkid123** [comment]
@tahmidefaz @justinorringer Thanks for checking this over. I'll update the config files.

And yeah I was a bit confused about the missing actions. I am glad that I did not misrepresent the miro board.

**Comment by tahmidefaz** [comment]
> > ```
> > * Saw a number of steps missing from what we have on Miro. Any reason why?
> > ```
> 
> Looks like they canned the fun one (its half baked anyway). He chose the right one

Why would they keep it there if they deprecated it... :facepalm: 
Thank you for pointing it out!

**Pull request merged**